README.md

```diff
@@ -56,8 +67,8 @@ $ chmod +x kops-linux-amd64                 # Add execution permissions
 $ mv kops-linux-amd64 /usr/local/bin/kops   # Move the kops to /usr/local/bin
 ```
 
-### Developer From Source
 
+### Developer From Source
```

Comment:
It's already [in the developer docs](/kubernetes/kops/tree/master/docs/install.md), I'll remove the section altogether.